packages/cli/src/services/orchestration/main/orchestration.main.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Service } from 'typedi';
+import { OrchestrationService } from '../../orchestration.base.service';
+
+@Service()
+export class OrchestrationMainService extends OrchestrationService {
+	sanityCheck(): boolean {
+		return this.initialized && this.isQueueMode && this.isMainInstance;
+	}
+
+	async getWorkerStatus(id?: string) {
+		if (!this.sanityCheck()) return;
+		await this.redisPublisher.publishToCommandChannel({
+			command: 'getStatus',
+			targets: id ? [id] : undefined,
+		});
+	}
+
+	async getWorkerIds() {
+		if (!this.sanityCheck()) return;
+		await this.redisPublisher.publishToCommandChannel({
+			command: 'getId',
+		});
+	}
+
+	async broadcastRestartEventbusAfterDestinationUpdate() {
+		if (!this.sanityCheck()) return;
+		await this.redisPublisher.publishToCommandChannel({
+			command: 'restartEventBus',
+		});
+	}
+
+	async broadcastReloadExternalSecretsProviders() {
```

Comment:
Not a big deal, but when setting up external secrets for the first time, when I click save n8n under the hood does:
- save
- enable

This basically triggers this twice. Would be good if we could prevent this but not a deal breaker if too complciated. Maybe a debounce?

Reply:
added debouncing on the handler level